Integrations/Autodesk/maya/scripts/unityCommands.mel

```diff
@@ -1028,4 +1050,35 @@ global proc unityExportModelAnim(){
 global proc unityCreateExportSet(){
     global int $UnityExportModelAnim;
     createExportSetDialog($UnityExportModelAnim);
+}
+
+global proc unityUpdateReferencedSetNamespace(){
```

Comment:
this is the only new thing I added in this file that isn't just rearranging functions